### PR TITLE
Changed name of Long handed inserter to Long-handed inserter, so it is aligned with the name in game

### PR DESCRIPTION
**Pull request by MortenTobiasNielsen**

### CODE REVIEW

File: cMain.cpp [hunk 1]
```diff
@@ -2332,6 +2337,11 @@ void cMain::OnGenerateScript(wxCommandEvent& event) {
 	size_t amount_of_tasks = tasks_data_to_save.size();
 
 	for (int i = 0; i < amount_of_tasks; i++) {
+		
+		if (i > 2937) {
+			int test = 1;
```
Comment (theis999) [inline cMain.cpp:2342]: This looks suspect. Is it some sort of debug code?

File: cMain.cpp [hunk 2]
```diff
@@ -2520,49 +2530,65 @@ void cMain::OnGenerateScript(wxCommandEvent& event) {
...
-	saver.open(generate_code_folder_location + "\\steps.lua");
+			saver.close();
+
+			saver.open(generate_code_folder_location + "\\info.json");
+			saver << "\{\n\t\"name\": \"" + generate_code_folder_location.substr(generate_code_folder_location.rfind("\\") + 1) + "\",";
+			saver << "\n\t\"version\": \"" << software_version << "\",";
+			saver << "\n\t\"title\": \"" + generate_code_folder_location.substr(generate_code_folder_location.rfind("\\") + 1) + "\",";
+			saver << info;
+			saver.close();
+		}
```
Comment (theis999) [inline cMain.cpp:2569]: I don't understand what you are trying to achieve here?

File: cMain.cpp (hunk 1 above)
Comment (MortenTobiasNielsen) [reply cMain.cpp:2342]: Yeah it is. :D

File: cMain.cpp (hunk 2 above)
Comment (MortenTobiasNielsen) [reply cMain.cpp:2569]: "// If the file is send to another person the folder location won't exist and should be set to something else."

**Comment by MortenTobiasNielsen** [comment]
I merge master in and try to solve the problem again, so it works with the current code.

**Review by theis999 — APPROVED** [review]
Looks good

**Pull request merged**